P202.Training.Test/RoleServiceTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Moq;
+using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace P202.Training.Test
+{
+    public class RoleServiceTest
+    {
+        [Fact]
+        public void RoleService_CreateData()
+        {
+            var newRole = new Role();
+            var mockpersonRepository = new Mock<IRoleRepository>();
+            mockpersonRepository.Setup(x => x.CreateRole(newRole)); 
+            mockpersonRepository.Object.CreateRole(newRole);
+            mockpersonRepository.Verify(x => x.CreateRole(newRole), Times.Once());
+        }
+
+        [Fact]
+        public void RoleService_DeleteData()
+        {
+            var deleteRole = new Role();
+            var mockpersonRepository = new Mock<IRoleRepository>();
+            mockpersonRepository.Setup(x => x.DeleteRole(deleteRole));
+            mockpersonRepository.Object.DeleteRole(deleteRole);
+            mockpersonRepository.Verify(x => x.DeleteRole(deleteRole), Times.Once());
+        }
```

Comment:
What are we testing here?

Reply:
I'm modifying this code because it's incomplete